pkg/pki/x509/x509.go

```diff
@@ -188,6 +188,13 @@ func (k PublicKey) EmailAddresses() []string {
 				names = append(names, strings.ToLower(name))
 			}
 		}
+		for _, name := range cert.URIs {
+			validate := validator.New()
```

Comment:
also, should we create `validate` once rather than churn objects across multiple loops?